src/index.ts

```diff
@@ -154,4 +157,20 @@ module.exports = class ServerlessPlugin {
       });
     }
   }
+
+  /**
+   * Tags the function(s) with plugin version
+   */
+  private async addPluginTag() {
+    this.serverless.cli.log(`Adding Plugin Version ${version}`);
+
+    this.serverless.service.getAllFunctions().forEach((functionName) => {
+      const functionDefintion: ExtendedFunctionDefinition = this.serverless.service.getFunction(functionName);
+      if (!functionDefintion.tags) {
+        functionDefintion.tags = {};
+      }
+
+      functionDefintion.tags[TagKeys.Plugin] = `v${version}`;
```

Comment:
don't think it's a big deal, but I think the entire tag string is not allowed to start with digit but the tag value i allowed to start with a digit (another common example is status_code:200) https://a.cl.ly/KourX2vN

Reply:
Interesting. While testing with just the semver, I noticed we were stripping out the **first digit**.

 ![](https://p-qKFgO2.t2.n0.cdn.getcloudapp.com/items/qGuNLeod/Image%202020-09-14%20at%2012.15.35%20PM.png?v=bf3e553ce213d88067167ecfd7b619de)

So this might be a bug on our end. @DarcyRaynerDD mentioned that we could fix the parsing function, but then it wouldn't be consistent with the lambda tags crawler. So I decided to avoid that rabbit hole by adding a `v` prefix